frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterActionDropdown.tsx

```diff
@@ -412,7 +412,7 @@ export function ClusterActionDropdown(props: { cluster: Cluster; isKebab: boolea
             {
                 id: 'ai-scale-up',
                 text: t('managed.ai.scaleUp'),
-                click: () => setScaleUpModalOpen(true),
+                click: (cluster: Cluster) => setScaleUpModalOpen(cluster),
```

Comment:
IIUC just a cluster name is needed here. Would be nice to limit the state to just that instead of whole cluster.

Reply:
done